package.json

```diff
@@ -73,6 +73,7 @@
     "path-to-regexp": "^0.1.2",
     "performance-now": "^2.1.0",
     "protobufjs": "^6.9.0",
+    "resolve": "^1.20.0",
```

Comment:
@bengl This is added as a dependency here but a devDependency in `LICENSE-3rdparty.csv`. Where is it actually used?

Reply:
https://github.com/DataDog/dd-trace-js/blob/e3766edae089944859941285d110425185ff32ec/packages/datadog-plugin-cucumber/src/index.js#L2

Reply:
Then `LICENSE-3rdparty.csv` should be updated accordingly.

Reply:
Fixed in https://github.com/DataDog/dd-trace-js/pull/1457